code/game/objects/items/weaponry.dm

```diff
@@ -266,6 +266,20 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	. = ..()
 	ADD_TRAIT(src, TRAIT_NODROP, CURSED_ITEM_TRAIT)
 
+//statistically similar to e-cutlasses
+/obj/item/katana/cutlass
```

Comment:
this seems like a weird thing to subtype off, katanas have some snowflake stuff like being able to fit in fedoras, id just make it its own thing or subtype off a claymore or something